src/goDocumentation.ts

```diff
@@ -97,11 +97,12 @@ function _devendorPkgs(pkgs: string[]): string[] {
 function _showDocumentationForPackage(pkg: string): void {
 	let uri = vscode.Uri.parse('godocumentation://').with({
 		path: pkg,
+		query: 'now=' + new Date().getTime().toString(),
```

Comment:
@ramya-rao-a is there a better way to bust vscode's text document content provider cache?